app/src/main/java/org/mozilla/focus/fragment/AddToHomescreenDialogFragment.java

```diff
@@ -62,7 +62,7 @@ public AlertDialog onCreateDialog(Bundle bundle) {
         final View dialogView = inflater.inflate(R.layout.add_to_homescreen, null);
         builder.setView(dialogView);
 
-        final Bitmap icon = IconGenerator.generateLauncherIcon(getActivity(), url);
+        final Bitmap icon = IconGenerator.Companion.generateLauncherIcon(getActivity(), url);
```

Comment:
Is there any particular reason why you didn't use @JvmStatic? Somehow this "Companion" object looks weird in Java world. However once more and more code becomes Kotlin code this will go away and then we might have a bunch of @JvmStatic annotations that don't make sense anymore? :)

Reply:
Because I was unaware of it! :)

It'd be pretty easy to `sed` @jvmstatic out of existence in any case. :)